src/engines/kotor/gui/options/gameplay.cpp

```diff
@@ -127,8 +167,20 @@ void OptionsGameplayMenu::updateDifficulty(int difficulty) {
 	}
 }
 
+void OptionsGameplayMenu::updateCheckbox(bool& option, const char* widget, bool state) {
```

Comment:
That should be `bool &option` and `const char *widget`, i.e. the `*` and `&` should hug the name, not the type.